DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -451,6 +463,7 @@ internal DiscordGuild()
         {
             this.Roles = new ReadOnlyConcurrentDictionary<ulong, DiscordRole>(this._roles);
             this.Emojis = new ReadOnlyConcurrentDictionary<ulong, DiscordEmoji>(this._emojis);
+            this.Stickers = new ReadOnlyDictionary<ulong, DiscordMessageSticker>(this._stickers);
```

Comment:
Great job.

Reply:
This should be ReadOnlyConcurrentDictionary.